packages/select/select.scss

```diff
@@ -93,6 +93,8 @@ $chevron-weight: rem(1px);
         text-overflow: ellipsis;
         white-space: nowrap;
         background-color: transparent;
+        color: inherit;
+        box-shadow: inset 0 -0.125rem 0 0 currentColor;
```

Comment:
Her kan du fint bruke `underline-color` som tidligere:
```scss
@include underline-color(currentColor, $border-color--focus);
```

Reply:
Jeg ser forresten nå at med `currentColor` blir linjen grå når det ikke er valgt en verdi (fordi `currentColor` er tekstfargen på elementet). Jeg kommer ikke på noen enkel måte å fikse dette på uten å lage en egen inverted-variant av stilen, dessverre.

Reply:
@piofinn Skal fargen være grå? Er det et bevisst design-valg?

Reply:
@Saegrov Det er i hvert fall tegnet sånn i biblioteket, både for tekstfeltene og for nedtrekkslistene. Meningen er formodentlig å synliggjøre når det ikke er tatt et valg.